resources/docs/simulation/README.md

```diff
@@ -10,9 +10,11 @@ You can use HASH's tools to build rich, detailed, agent-based simulations of [fa
 
 ## Getting Started
 
-{% hint style="success" %}
+<Hint style="success">
```

Comment:
This might look odd but throughout the MD files, we need to add a "new line" after each MDX tag so that the MD below it renders correctly.

This continues for <Tabs>, <TextTabs>, etc